### PR TITLE
1587 sizewarnings

### PR DESCRIPTION
**Pull request by vidya-v**
VIC Appliance Checklist:
- [ ] Up to date with `master` branch
- [ ] Added tests
- [ ] Considered impact to upgrade
- [ ] Tests passing
- [ ] Updated documentation
- [ ] Impact assessment checklist

If this is a feature or change to existing functionality, consider areas of impact with the [Impact
Assessment Checklist](https://github.com/vmware/vic-product/blob/master/installer/docs/CHANGE.md)

Fixes #

### CODE REVIEW

**Comment by vidya-v** [comment]
Added information about reconfiguring alerts to avoid size related warnings that might occur during docker pull.
@zjs , @stuclem, request your review.

Thanks!

File: docs/user_doc/vic_vsphere_admin/vch_compute.md [hunk 1]
```diff
@@ -217,7 +217,9 @@ Specify a value of greater than 1. If not specified, `vic-machine create` sets t
 
 The amount of memory for the VCH endpoint VM. Set this option to increase the amount of memory in the VCH endpoint VM if the VCH will pull large container images.
 
-**NOTE** With the exception of VCHs that pull large container images, increase the overall amount of memory for the VCH resource pool, rather than the amount of memory of the VCH endpoint VM. Use `docker create -m` to set the memory on container VMs. This option is mainly intended for use by VMware Support.
+Image layers are stored in memory during `docker pull`. If you have enabled or inherited memory usage alerts on the VCH endpoint and you download a large VMFS virtual disk (VMDK), you might trigger a memory usage alert. You might want to reconfigure the alerts to avoid receiving warnings related to size.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/vch_compute.md]: VCH endpoint ==> VCH endpoint VM

download **an image with** a large VMFS virtual disk (VMDK)

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/vch_compute.md]: Thanks, @stuclem . Fixed them

**Comment by vidya-v** [comment]
Thanks, @zjs !

**Pull request merged**